build.gradle

```diff
@@ -157,8 +157,9 @@ task buildDockerContainer {
 }
 
 task startDockerContainer(type: Exec) {
-    commandLine "bash", "-c", "docker run -e \"NAKADI_OAUTH2_ENABLED=false\" -e \"NAKADI_CONFIG=file:////envs/development.conf\" " +
-            "--net container:local-kafka -i -t -d " +
+    commandLine "bash", "-c", "docker run -e \"NAKADI_OAUTH2_ENABLED=false\" -e \"ZK_BROKERS=127.0.0.1:2181\" " +
```

Comment:
I think it would be better if default local configuration will be stored in application.yml. And that it can be overridden with environment variables.
And `@Value` annotation is a perfect tool for that. For instance if you have 
```java
@Value("${zookeeper.brokers}")
private String zookeeperBrokers;
```
Spring will first look for environment variable `ZOOKEEPER_BROKERS` if it doesn't find it - it will look for setting `zookeeper.brokers` in application.yml.
So for local config we will have everything in application.yml and for real deployment we will specify everything with env vars.